spec/models/mark_spec.rb

```diff
@@ -50,8 +43,8 @@
       FactoryBot.create(:rubric_mark, mark: 4)
     end
     it 'equals to mark times weight' do
-      markable = RubricCriterion.find(rubric_mark.markable_id)
-      expect(rubric_mark.mark).to eq(markable.weight)
+      related_rubric = RubricCriterion.find(rubric_mark.criterion_id)
```

Comment:
Let's simplify this query here. You can do `related_rubric = rubric_mark.criterion`.